src/orion/algo/evolution_es.py

```diff
@@ -243,9 +243,19 @@ def get_candidates(self, rung_id):
         return points
 
     def _mutate(self, winner_id, loser_id):
-        self.mutate_func(self, winner_id, loser_id,
-                         self.mutate_attr["multiply_factor"],
-                         self.mutate_attr["add_factor"])
+        select_genes_key_list = np.random.choice(self.search_space_remove_fidelity,
+                                                 self.eves.nums_mutate_gene,
+                                                 replace=False)
+        self.copy_winner(winner_id, loser_id)
+        for i, _ in enumerate(select_genes_key_list):
+            space = self.space.values()[select_genes_key_list[i]]
+            old = self.eves.population[select_genes_key_list[i]][loser_id]
+
+            new = self.mutate_func(self.mutate_attr["multiply_factor"],
```

Comment:
I believe the mutate_attr should be passed as `**kwargs` arguments to support any type of function and arguments. I can understand why space and old arguments would be required for any function. Is the volatility more a parameter of the general evolutionary algorithm or more a parameter of the mutation function?

Reply:
Good suggestion. I have done it. Please check it. Volatility is more a mutation parameter, so I  view it as other mutate_attr.